js/content/community.js

```diff
@@ -858,6 +878,58 @@ let ProfileHomePageClass = (function(){
     return ProfileHomePageClass;
 })();
 
+
+let GroupHomePageClass = (function(){
+
+    function GroupHomePageClass() {
+        this.addExtraLinks();
+    }
+
+    GroupHomePageClass.prototype.addExtraLinks = function() {
```

Comment:
Since the links to profiles on other pages like SteamRep for example are called profile links, I'd call these group links
```suggestion
    GroupHomePageClass.prototype.addGroupLinks = function() {
```